packages/react-next/src/components/FocusTrapZone/FocusTrapZone.test.tsx

```diff
@@ -704,44 +720,41 @@ describe('FocusTrapZone', () => {
   });
 
   describe('Nested FocusTrapZones Stack Behavior', () => {
-    function getFocusStack(): FocusTrapZone[] {
-      // eslint-disable-next-line @typescript-eslint/no-explicit-any
-      return (FocusTrapZone as any)._focusStack;
+    function getFocusStack(): string[] {
+      return FocusTrapZone.focusStack;
     }
 
     beforeAll(() => {
       getFocusStack().length = 0;
     });
 
     it('FocusTrapZone maintains a proper stack of FocusTrapZones as more are mounted/unmounted.', async () => {
-      let focusTrapZoneFocusStack: FocusTrapZone[] = getFocusStack();
-      const topLevelDiv = (ReactTestUtils.renderIntoDocument(
-        <div>
-          <FocusTrapZoneTestComponent />
-        </div>,
-      ) as unknown) as HTMLElement;
-
-      const buttonA = topLevelDiv.querySelector('.a') as HTMLElement;
-
-      const buttonB = topLevelDiv.querySelector('.b') as HTMLElement;
+      let focusTrapZoneFocusStack: string[] = getFocusStack();
+
+      let topLevelDiv: HTMLElement;
+      ReactTestUtils.act(() => {
+        topLevelDiv = (ReactTestUtils.renderIntoDocument(
+          <div>
+            <FocusTrapZoneTestComponent />
+          </div>,
+        ) as unknown) as HTMLElement;
+      });
+      const buttonA = topLevelDiv!.querySelector('.a') as HTMLElement;
+      const buttonB = topLevelDiv!.querySelector('.b') as HTMLElement;
 
       expect(focusTrapZoneFocusStack.length).toBe(2);
-      const baseFocusTrapZone = focusTrapZoneFocusStack[0];
```

Comment:
Did the behavior change such that this test didn't work, or was this just not necessary to cover here? (same for other changes in this test)